main.go

```diff
@@ -97,6 +98,15 @@ func main() {
 	}
 	setupLog.Info("loaded configuration successfully", "config", conf.Get())
 
+	planner := pln.New(pln.InstanceConfiguration{
+		GlobalConfig: conf.Get(),
+	}, nil)
+	if _, err := planner.NodeSelector(); err != nil {
+		setupLog.Error(err, "invalid node selector configuration value",
+			"note", "value must be a JSON object containing strings")
+		os.Exit(1)
+	}
+
```

Comment:
Should we place the above validation block before LINENO: 99 such that we say "loaded configuration successfully" after checking `DefaultNodeSelector` too?

Reply:
The first one indicates that it successfully got a configuration. This new one is saying a value in the config is no good. Let's wait on it and change it if we get users who complain.

Reply:
Ok.